src/scripts/components/Header.jsx

```diff
@@ -68,9 +72,9 @@ export default class Header extends React.Component {
                 className="header__stamp"
                 alt="Resound Store logo"
               />
-              <Link to="/" className="header__link">
```

Comment:
I think my question of why use a click event instead of a link related to my other question above.

Reply:
`Link` is a react router component that lets users navigate back to a specific route. It does not reload the react application state. I changed it to the span because I want to trigger an actual "navigate to a different page" action so that the application gets re-initialized.